config/database.yml

```diff
@@ -1,23 +1,23 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  username: datahub
-  password: datahub
-  host: localhost
+  username: <%= ENV.fetch("DATABASE_USERNAME") { "datahub" } %>
```

Comment:
`ENV['DATABASE_USERNAME'] || 'datahub'` is the standard ruby idiom.

Reply:
Thanks! I was following another example https://github.com/slovensko-digital/harvester.ecosystem/blob/master/config/database.yml#L9